src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+const ADD_BOOK = 'ADD_BOOK';
+const DELETE_BOOK = 'DELETE_BOOK';
```

Comment:
- [ ] The value of  action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). So it will be something like:
``` javascript
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  ...
```
Kindly change it

Reply:
This issue is well resolved! ✔️